server/internal/config_parser.go

```diff
@@ -48,6 +48,7 @@ type GridAccount struct {
 	Network   string `json:"network"`
 }
 
+// Salt struct to hold salt for hashing
 type Salt struct {
 	Salt string `json:"salt"`
```

Comment:
Why did we create a separate struct for the salt? it can be safely part of the configurations struct directly `salt string`